fastlane/lanes/release.rb

```diff
@@ -364,10 +512,17 @@ def release_notes_short_paths
 
   def bundle_file_path(app, version)
```

Comment:
Similar to a previous comment, should we rename `version` to `version_name`? I think we want to do this everywhere we were using `version['name']` because otherwise it's going to be very confusing what `version` refers to.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/commit/6ab18efe9261a63b77766614c06c3bc661a697f6